### ISSUE TITLE
Push Notifications should follow Github Settings

### ISSUE THREAD

**Issue by brad-decker** [thedillonb/CodeHub#189]
#### Currently:
I watch repositories that I want to keep track of but have my personal github settings to never send me an email unless i'm @mentioned or if i'm involved in the conversation in some way. This way i can stay up to date with repositories i'm interested in but don't have to deal with a huge inbox. 

CodeHub - great app not replaceable for me - however sends me a push notification for everything which gets annoying especially if i forget to turn my phone on silent. 

#### Desired: 
CodeHub only triggers a push notification if the alert would have generated an email via Github.

**Comment by christopherthielen** [comment]
I have to disagree with this. I don't need a push notification for a github email, because I already get an email notification.  

I like the current behavior where I get all notifications pushed, but only emails for conversations I have participated in.

**Comment by brad-decker** [comment]
It makes it impossible for me to use notifications on this app. I have my
email preferences set to "only if mentioned" so that I can watch
repositories and not get an email for every PR / issue / comment etc. this
way I can stay up to date on things like React and Node, without thousands
of emails. Regardless of my settings I still get a push notification for
everyone of them making me either have to change how I use GitHub (not
gonna happen) or stop using notifications on this app.

**Comment by mikeerickson** [comment]
I would agree, the current method CodeHub issues notifications is perfect.
